src/openrct2/localisation/Localisation.cpp

```diff
@@ -502,7 +502,7 @@ money32 string_to_money(const char* string_to_monetise)
     auto whole = static_cast<int32_t>(number);
     auto fraction = static_cast<uint8_t>(ceil((number - whole) * 100.0));
 
-    money32 result = MONEY(whole, fraction);
+    money32 result = (whole)*10 + ((fraction) / 10);
```

Comment:
I guess there are two approaches here:
- Reinstate the MONEY define/constexpr
- Just multiply/divide `number` by the conversion rate and feed the result into `result`, skipping the splitting out of `whole` and `fraction`.

Personally, I'm gravitating towards the latter. @duncanspumpkin what do you think?

Reply:
Yes scrap the whole, fraction just multiply by the factor 10.

Reply:
@duncanspumpkin How would we want to check for MONEY overflow then, as a float?

Reply:
You've two options. Either do `uint64_t amount = std::min<uint64_t>(amountFloat *10.0, std::limits<uint32>::max()` or do `floatAmount = std::min<double>(floatMount*10, std::limits<uint32>::max())`.
In general we try to minimise the amount of time anything is a floating point so I'd go with the first option. Then of course cast the final result back to uint32.